index.js

```diff
@@ -138,6 +136,10 @@ PSPDFKitView.propTypes = {
      * Controls wheter or not the default action for tapped annotations is processed. Defaults to processing the action (false).
      */
     disableDefaultActionForTappedAnnotations: PropTypes.bool,
+    /**
+     * Controls whether or not the document will be automatically saved. Defaults to automatically saving (false).
+     */
+    disableAutomaticSaving: PropTypes.bool,
```

Comment:
The Android version of this flag wasn't merged yet.

Reply:
I’ll add the iOS flag before merging this PR.